src/Struct/Config.php

```diff
@@ -159,7 +159,6 @@ private function initializeReadonlyConfig(?string $salesChannelId): void
     }
 
     /**
-     * @param mixed $default
```

Comment:
That docblock information shouldn't get lost. Otherwise the IDE assumes that you can only add `null` here, which is not the case.

Reply:
I think this was done by the linter. But OK I will revert this.